src/Common/src/Interop/OSX/System.Native/Interop.ProtocolStatistics.cs

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        public unsafe struct TcpGlobalStatistics
+        {
+            public readonly ulong ConnectionsAccepted;
+            public readonly ulong ConnectionsInitiated;
+            public readonly ulong CumulativeConnections;
+            public readonly ulong ErrorsReceived;
+            public readonly ulong FailedConnectionAttempts;
+            public readonly ulong SegmentsReceived;
+            public readonly ulong SegmentsResent;
+            public readonly ulong SegmentsSent;
+            public readonly int CurrentConnections;
+            private readonly int __padding;
```

Comment:
I just used the double underscore as a signal to "not use this". The fact that it's also private means it probably doesn't matter what the field is called.

The padding is necessary because clang wants structs to be aligned strictly, or some-such (we have strict warnings-as-errors on). I'm not terribly familiar with what the exact warning or danger about not following it is.